docker/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM debian:jessie
+
+ARG DEBIAN_FRONTEND="noninteractive"
+
+ENV LANG C.UTF-8
```

Comment:
I have a feeling that the locale isn't set up in the base debian image. Not including this causes locale errors to be reported during installation of some packages (especially true for postgresql, though that isn't installed in this container).